src/internal/collection/postgres_listener.go

```diff
@@ -217,6 +217,7 @@ type postgresEvent struct {
 	protoData []byte          // the serialized protobuf of the row data (exclusive with storedID)
 	storedID  string          // the ID of a temporary row in the large_notifications table
 	err       error           // any error that occurred in parsing
+	create    bool
```

Comment:
Is this used anywhere?

Reply:
no reverting this. Was using this in the previous logic iteration